sdk/elixir/runtime/main.go

```diff
@@ -142,25 +140,34 @@ func (m *ElixirSdk) WithNewElixirPackage(ctx context.Context, modName string) *E
 }
 
 // Generate the SDK into the container.
-func (m *ElixirSdk) WithSDK(introspectionJson *File) *ElixirSdk {
+func (m *ElixirSdk) WithSDK(introspectionJson *dagger.File) *ElixirSdk {
 	if m.err != nil {
 		return m
 	}
 	m.Container = m.Container.
-		WithDirectory(genDir, m.SDKSourceDir, ContainerWithDirectoryOpts{
+		WithDirectory(genDir, m.SDKSourceDir, dagger.ContainerWithDirectoryOpts{
 			// Excludes all unnecessary files from official SDK.
 			Exclude: []string{
+				"README.md",
+				"CHANGELOG.md",
+				".changes",
+				"changie.yaml",
+				".credo.exs",
 				"dagger_codegen",
+				"test",
 				// We'll do generate code on the next step.
 				"lib/dagger/gen",
+				// Now the runtime is in the root of the SDK. We need
+				// to exclude them.
+				"dagger.json",
 				"runtime",
```

Comment:
This is probably not necessary. The `SDKSourceDir` should already include only what you need for this step.

Reply:
Fixed at https://github.com/dagger/dagger/pull/7667#discussion_r1685321338.

Reply:
> Fixed at [#7667 (comment)](https://github.com/dagger/dagger/pull/7667#discussion_r1685321338).

That was just `dagger.json`, but I actually meant all of these exclude patterns. There are things here that are not in `dagger.json`'s "include", so no need to exclude what's not in the dir.